app/src/main/res/layout/activity_request_detail.xml

```diff
@@ -11,133 +11,133 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent">
 
-    <androidx.appcompat.widget.AppCompatTextView
-        android:id="@+id/tvOtherUserName"
-        android:layout_width="0dp"
-        android:layout_height="wrap_content"
-        android:layout_marginEnd="16dp"
-        android:layout_marginStart="16dp"
-        android:layout_marginTop="16dp"
-        android:textAlignment="textEnd"
-        android:textSize="28sp"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintHorizontal_bias="0.957"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        tools:text="Lola Zink" />
+        <androidx.appcompat.widget.AppCompatTextView
+            android:id="@+id/tvOtherUserName"
```

Comment:
I'm not sure whether formatting the xml layouts was part of the issue. Have you used the inbuilt formatter in android studio?

Reply:
The formatting of the XML files were not required but the inbuilt formatter did those changes.

Reply:
> I'm not sure whether formatting the xml layouts was part of the issue. Have you used the inbuilt formatter in android studio?

I have successfully removed all the extra formatting of the xml layouts. 
The removal of tools:text part is only left now.